web/src/components/tree/KotsApplicationTree.jsx

```diff
@@ -85,8 +85,11 @@ class KotsApplicationTree extends React.Component {
         <Helmet>
           <title>{`${this.props.app?.name} Files`}</title>
         </Helmet>
-
-        <div className="edit-files-banner u-fontSize--small u-fontWeight--medium">Need to edit these files? <span onClick={this.toggleInstructionsModal} className="replicated-link">Click here</span> to learn how</div>
+        {!this.props.isHelmManaged &&
+          <div className="edit-files-banner u-fontSize--small u-fontWeight--medium">
+            Need to edit these files? <span onClick={this.toggleInstructionsModal} className="replicated-link">Click here</span> to learn how
+          </div>
+        }
```

Comment:
Should the file viewer shift up a bit when the link isn't there rather than just removing the link? Seems like more whitespace than I'd expect and compared to other tabs.